infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesWorkspaceServiceAccount.java

```diff
@@ -82,6 +87,16 @@ void prepare() throws InfrastructureException {
         .kubernetesRoleBindings()
         .inNamespace(namespace)
         .createOrReplace(createViewRoleBinding());
+
+    // If the user specified an additional cluster role for the workspace,
+    // create a role binding for it too
+    if (!isNullOrEmpty(this.clusterRoleName)) {
+      k8sClient
+          .rbac()
+          .kubernetesRoleBindings()
+          .inNamespace(namespace)
+          .createOrReplace(createCustomRoleBinding(this.clusterRoleName));
```

Comment:
What does an error look like when `this.clusterRoleName` does not exist on the cluster? I'm mostly worried about troubleshooting in case of a typo.

Reply:
@amisevsk Good point! Let me add a quick log message in the event the clsuter role doesn't exist.